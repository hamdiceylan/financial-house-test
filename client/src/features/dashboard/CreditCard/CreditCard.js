import React from 'react'
import { Grid, Header } from 'semantic-ui-react'

const CreditCard = ({client}) => {
  return (
    <Grid.Column width={8}>
        <Header icon='payment' content='Card Details'/>
        <p>Card number: <strong>{client.number}</strong></p>
        <p>Expiry Month: <strong>{client.expiryMonth}</strong></p>
        <p>Expiry Year: <strong>{client.expiryYear}</strong></p>
    </Grid.Column>
  )
}

export default CreditCard
