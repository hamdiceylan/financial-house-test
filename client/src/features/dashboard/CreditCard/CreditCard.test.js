import React from 'react'
import { shallow } from 'enzyme'
import CreditCard from './CreditCard';


describe('CreditCard', () =>{
    const props = {
        client : {
            number: '4242-4242-4242-4242',
            expiryMonth: '12',
            expiryYear: '21'
        }
    }
    const creditCard = shallow(<CreditCard {...props} />);

    it('renders correctly', () => {
        expect(creditCard).toMatchSnapshot();
    })

})

