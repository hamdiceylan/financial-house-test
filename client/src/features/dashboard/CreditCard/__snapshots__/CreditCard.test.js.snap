// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditCard renders correctly 1`] = `ShallowWrapper {}`;
